.github/workflows/ci.yml

```diff
@@ -63,6 +63,8 @@ jobs:
           - 2d_part1
           - 2d_part2
           - 2d_part3
+          - 2d_simplices
+          - 3d_simplices          
```

Comment:
Remember to update `.codecov.yml` accordingly

Reply:
Where in `.codecov.yml`? I don't see any other test names.

Reply:
Sorry, I should have been more clear - this is more a note to myself.

Long form: The number of tests changes, so we need to adapt https://github.com/trixi-framework/Trixi.jl/blob/f8f138e6d0acec9d5297dab0c7a27ff2d89dae14/.codecov.yml#L3-L14

Reply:
Ah, gotcha. Thanks!